packages/daf-resolver/src/__tests__/resolver.test.ts

```diff
@@ -1,14 +1,18 @@
 import { DafResolver } from '../resolver'
+import { Resolver } from 'did-resolver'
 
 describe('daf-resolver', () => {
   it('should throw error when misconfigured', () => {
     expect(() => {
-      new DafResolver({})
+      new DafResolver({
+        //@ts-ignore
+        resolver: undefined
+      })
     }).toThrow()
   })
 
```

Comment:
We should have some more tests here, even as TODO
```suggestion
it.todo('should resolve example did')
it.todo('should fail predictably when unsupported method is resolved')
it.todo('should resolve ethr-did with RPC URL')
it.todo('should resolve ethr-did with custom web3 provider')
```